packages/cli/src/api/catalog.test.ts

```diff
@@ -147,6 +147,24 @@ describe("Catalog", function () {
       expect(messages).toMatchSnapshot()
     })
 
+    it("should extract only files passed on options", function () {
+      const catalog = new Catalog(
+        {
+          name: "messages",
+          path: "locales/{locale}",
+          include: [fixture("collect/componentA")],
```

Comment:
Shouldn't we also add another directory to test that the `files` actually filter available files? When I comment out line `163`, the test passes anyway. Adding an extra file or directory, the snapshot should stay the same.

```suggestion
          include: [fixture("collect/componentA"), fixture("collect/componentB.js")],
```